app/src/main/java/eu/kanade/presentation/manga/components/MangaNotesDisplay.kt

```diff
@@ -0,0 +1,60 @@
+package eu.kanade.presentation.manga.components
+
+import androidx.compose.animation.animateContentSize
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.text.selection.SelectionContainer
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import com.mohamedrejeb.richeditor.model.rememberRichTextState
+import com.mohamedrejeb.richeditor.ui.material3.RichText
+
+@Composable
+fun MangaNotesDisplay(
+    content: String,
+    modifier: Modifier,
+) {
+    val alpha = remember { Animatable(1f) }
+    var isFirstUpdate by remember { mutableStateOf(true) }
+
+    val primaryColor = MaterialTheme.colorScheme.primary
+    val richTextState = rememberRichTextState()
```

Comment:
```suggestion
    val richTextState = rememberRichTextState()
    val primaryColor = MaterialTheme.colorScheme.primary
```
Move this before `alpha`